### PR TITLE
Feature/#200 create docs for components

### PR DESCRIPTION
**Pull request by Rahul139-stack**
Added Markdown docs for map component
[Readme link](https://github.com/typerefinery-ai/typerefinery-websight/tree/feature/%23200-create-docs-for-components/application/backend/src/main/resources/apps/typerefinery/components/widgets/map)

### CODE REVIEW

File: application/backend/src/main/resources/apps/typerefinery/components/widgets/map/image/image.png
Comment (Harikarthyk) [inline application/backend/src/main/resources/apps/typerefinery/components/widgets/map/image/image.png:1]: What is the purpose of this image ?

**Review by Harikarthyk — CHANGES_REQUESTED** [review]
Remove images that are not used, and along with that, please add some screenshots or a link to the readme.

Thanks

**Comment by wildone** [comment]
@Rahul139-stack is feeback updates completed?

**Comment by Rahul139-stack** [comment]
> @Rahul139-stack is feeback updates completed?

@wildone @Harikarthyk  Code updated

**Review by Harikarthyk — CHANGES_REQUESTED** [review]
image

In the above screenshots marker fields have misplaced values, please make them correct.

**Pull request closed without merging**